src/components/Popups/index.tsx

```diff
@@ -74,16 +72,18 @@ export default function Popups() {
           <PopupItem key={item.key} content={item.content} popKey={item.key} removeAfterMs={item.removeAfterMs} />
         ))}
       </FixedPopupColumn>
-      <MobilePopupWrapper height={activePopups?.length > 0 ? 'fit-content' : 0}>
-        <MobilePopupInner>
-          {activePopups // reverse so new items up front
-            .slice(0)
-            .reverse()
-            .map((item) => (
-              <PopupItem key={item.key} content={item.content} popKey={item.key} removeAfterMs={item.removeAfterMs} />
-            ))}
-        </MobilePopupInner>
-      </MobilePopupWrapper>
+      {activePopups?.length > 0 && (
```

Comment:
This change fixes artificial gap caused by `padding-top` when there are no popups:

<img width="498" alt="Screenshot 2022-12-15 at 14 25 10" src="https://user-images.githubusercontent.com/2464966/207952577-15cc67ab-faa0-4114-9786-def22c73f68f.png">

No changes when actual popup is displayed (note styling is off, but this is not caused by this PR):

<img width="465" alt="Screenshot 2022-12-15 at 14 34 32" src="https://user-images.githubusercontent.com/2464966/207952680-c50dd7cd-186b-4283-b0b5-19811a16d41a.png">